app/post/controllers/post-detail-controller.js

```diff
@@ -140,13 +142,19 @@ function (
         }
     });
 
+    $scope.deletePostRequested = function () {
+        // I feel there must be a prettier way to do this in angular/JS
+        $scope.userRequestedPostDelete = true;
+    };
+
+    $scope.cancelDelete = function () {
+        // again - feel there must be a more angular-ish way to do this
+        $scope.userRequestedPostDelete = false;
+    };
+
     $scope.deletePost = function () {
-        $translate('notify.post.destroy_confirm').then(function (message) {
-            if (window.confirm(message)) {
```

Comment:
Nope, cause this function now only gets called when the deletion is confirmed in the modal.